src/sentry/incidents/logic.py

```diff
@@ -598,13 +611,16 @@ def create_alert_rule(
             sensitivity=sensitivity,
             seasonality=seasonality,
             detection_type=detection_type,
-        )
+        )  # type: ignore[misc]
```

Comment:
should be fixed before shipping -- what's the bug here?

Reply:
wow I didn't even know django could do that

a typesafe alternative would be something like:


```python
class _OwnerKwargs(TypedDict):
    team_id: int | None
    user_id: int | None

def actor_to_alert_rule_owner_kwargs(actor: Actor | None) -> _OwnerKwargs: ...
```

and then splat that into the caller

Reply:
Nice one. Done: https://github.com/getsentry/sentry/pull/75541/commits/adf5a0d2864369bca9ca9ab567936b7cb5ae7de1